bin/wait_display

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+cnt=0
+max=120
```

Comment:
@FriggaHel is there any particular decision on waiting 2minutes? are we communicating this to the user while we wait?

Reply:
@vrunoa There is no specific communication to the user. In all my local (docker) testing, the wait it not felt since `xvfb` starts in less than a second. Resources available on Github Actions executor may be smaller, that make `xvfb` to start in 5sec-ish.

I put the 2min limit as an higher cap, but can be decreased to 10s I think it would be better.

Reply:
Do we want to echo the "wait"? Just wondering how a user will react to seeing nothing for 2min.

Reply:
@vrunoa Added messaging + decrease time-limit.